CHANGES.md

```diff
@@ -104,6 +104,13 @@ OpenSSL 3.1
 
    *Sergey Kirillov, Andrey Matyukov (Intel Corp)*
 
+ * The functions `OPENSSL_LH_stats` and `OPENSSL_LH_stats_bio` now only report
+   the `num_items`, `num_nodes` and `num_alloc_nodes` statistics. All other
+   statistics are no longer supported. For compatibility, these statistics are
+   still listed in the output but are now always reported as zero.
```

Comment:
As this is going to be merged to 3.0 as well this should go into the section `Changes between 3.0.2 and 3.0.3` which itself needs to be added.